docs/updating/migration-to-cdn/angular.md

```diff
@@ -0,0 +1,180 @@
+
+## Prerequisites
+
+Remove the existing CKEditor&nbsp;5 packages from your project. If you are using the NPM-based installation, you can remove it by running the following command:
+
+```bash
+npm uninstall ckeditor5 ckeditor5-premium-features
+```
+
+Upgrade the CKEditor&nbsp;5 Angular integration to the latest version. You can find the latest version in the [CKEditor&nbsp;5 Builds](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/frameworks/angular.html) documentation.
+
+Ensure that your testing suite uses real web browser environments for testing. If you are using `jsdom` or any other environment without a real DOM, you may need to adjust the testing suite configuration to use a real browser because CDN scripts injection might be not recognized properly in such environments.
+
+## Migration steps
+
+### Step 1: Remove CKEditor&nbsp;5 imports
+
+If you have any CKEditor 5 imports in your Vue components, remove them. For example, remove lines like:
+
+```javascript
+import { ClassicEditor, ... } from 'ckeditor5';
+import { EasyImage, ... } from 'ckeditor5-premium-features';
+```
+
+### Step 2: Update your Angular components to use CDN
+
+Replace the CKEditor&nbsp;5 NPM package imports with the CDN script imports and use the `loadCKEditorCloud` function to load the CKEditor&nbsp;5 scripts. The `loadCKEditorCloud` function is a part of the `@ckeditor/ckeditor5-angular` package and is used to load CKEditor&nbsp;5 scripts from the CKEditor Cloud service.
+
+**Before:**
+
+```ts
+// app.component.ts
+
+import { Component, ViewEncapsulation } from '@angular/core';
+import { CKEditorModule } from '@ckeditor/ckeditor5-angular';
+import { ClassicEditor, Bold, Essentials, Italic, Mention, Paragraph, Undo } from 'ckeditor5';
+import { SlashCommand } from 'ckeditor5-premium-features';
+
+@Component( {
+	selector: 'app-root',
+	templateUrl: './app.component.html',
+	styleUrls: ['./app.component.css'],
+	encapsulation: ViewEncapsulation.None,
+	imports: [ CKEditorModule ],
+	standalone: true
+} )
+export class AppComponent {
+	title = 'angular';
+
+	public Editor = ClassicEditor;
+	public config = {
+		licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
+		plugins: [ Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo ],
+		toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+		// mention: {
+		//     Mention configuration
+		// }
+	}
+}
+```
+
+**After:**
+
+```ts
+// app.component.ts
+
+import { Component, ViewEncapsulation } from '@angular/core';
+import { CKEditorModule } from '@ckeditor/ckeditor5-angular';
+
+@Component( {
+	selector: 'app-root',
+	templateUrl: './app.component.html',
+	styleUrls: ['./app.component.css'],
+	encapsulation: ViewEncapsulation.None,
+	imports: [ CKEditorModule ],
+	standalone: true
+} )
+export class AppComponent {
+	title = 'angular';
+
+	public Editor;
+	public config;
+
+	public ngOnInit() {
```

Comment:
You can stress with a comment in the code what changed, this way it will be easier to scan.

```js
// ADDED


```